package.json

```diff
@@ -203,36 +203,36 @@
   "bugs": "https://github.com/preactjs/preact/issues",
   "homepage": "https://preactjs.com",
   "devDependencies": {
-    "@actions/github": "^6.0.0",
+    "@actions/github": "^6.0.1",
     "@actions/glob": "^0.5.0",
-    "@babel/core": "^7.26.0",
-    "@babel/plugin-transform-react-jsx": "^7.25.9",
-    "@babel/plugin-transform-react-jsx-source": "^7.25.9",
-    "@babel/preset-env": "^7.26.0",
-    "@babel/register": "^7.25.9",
-    "@biomejs/biome": "^1.9.4",
+    "@babel/core": "^7.28.0",
+    "@babel/plugin-transform-react-jsx": "^7.27.1",
+    "@babel/plugin-transform-react-jsx-source": "^7.27.1",
+    "@babel/preset-env": "^7.28.0",
+    "@babel/register": "^7.27.1",
+    "@biomejs/biome": "^2.1.2",
     "@types/node": "^18.19.87",
-    "@vitest/browser": "^3.2.1",
+    "@vitest/browser": "^3.2.4",
     "@vitest/coverage-v8": "^3.2.4",
     "babel-plugin-transform-rename-properties": "0.1.0",
     "coveralls": "^3.1.1",
     "cross-env": "^7.0.3",
     "errorstacks": "^2.4.1",
-    "esbuild": "^0.24.0",
+    "esbuild": "^0.25.8",
     "husky": "^9.1.7",
     "kolorist": "^1.8.0",
     "microbundle": "^0.15.1",
     "npm-run-all2": "^7.0.0",
     "oxlint": "^1.8.0",
-    "playwright": "^1.53.1",
+    "playwright": "^1.54.1",
     "preact-render-to-string": "^6.5.0",
     "prop-types": "^15.8.1",
     "sade": "^1.8.1",
     "terser": "5.16.0",
     "typescript": "5.1.6",
     "undici": "^4.12.0",
     "vite": "^6.2.0",
```

Comment:
Vite has a 7.0.0 but it looks like it currently crashes due to `crypto.hash` not being a function locally. Volta does not seem to apply the node v20.19 for me 😅 going to leave the bump in the middle for now